qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -554,6 +554,9 @@
             test_login_guest = ssh ${guest_user}@%s -o stricthostkeychecking=no ls /home
         - check_get_cpustats:
             only Linux
+            only RHEL.9
+                no RHEL.9.1 RHEL.9.0
+            required_qemu = [7.1.0, )
```

Comment:
On ppc64le,
1.This test case will be cancelled for rhel8.x guest
2.I think we need to consider that the rhel 9.x guest only runs qemu-kvm-6.2.0 on ppc64le, 
   so we need to adjust it 
required_qemu = [7.1.0, )
 (3/3) Host_RHEL.m8.u9.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.3.0.ppc64le.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_get_cpustats: CANCEL: Got host qemu version:6.2.0-33, which is not in [7.1.0, ) (3.09 s)

Reply:
this is supported start from qemu7.1.0 so I think it's ok if we skip ppc since we only have QEMU-6.2 with it

Reply:
No, the qemu_guest_agent with higher version ( [7.1.0, ) can be installed to the rhel 9.x guest. We can't simply cancel the test for rhel9.x guest on rhel8.x host.